WindowsAzure/MediaServices/MediaServicesRestProxy.php

```diff
@@ -304,25 +304,28 @@ private function _getEntity($path)
     }
 
     /**
-     * Create entity list.
+     * Get entity list.
      *
-     * @param string $path REST path
+     * @param string $path        REST path
+     * @param array  $queryParams URL query parameters. An example is $queryParams = ['$top' => 20, '$skip' => 40] to support paging
      *
      * @return array Entities list data
      */
-    private function _getEntityList($path)
+    private function _getEntityList($path,  $queryParams = null)
```

Comment:
If you set the default value to `[]` you can omit the check inside the method body. It also should be type hinted to `array`. And please, remove the surplus blank space between arguments:
```php
private function _getEntityList($path, array $queryParams = [])
```